package.json

```diff
@@ -69,10 +69,10 @@
     "@babel/plugin-proposal-pipeline-operator": "^7.12.1",
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-react": "^7.12.10",
-    "@storybook/addon-storysource": "^5.3.19",
-    "@storybook/addons": "^5.3.19",
-    "@storybook/react": "^5.3.19",
-    "@storybook/source-loader": "^5.3.19",
+    "@storybook/addon-storysource": "^6.1.11",
+    "@storybook/addons": "^6.1.11",
+    "@storybook/react": "^6.1.11",
+    "@storybook/source-loader": "^6.1.11",
```

Comment:
Why not the latest version? I believe its 6.1.15

Reply:
Fixed version to be 6.1.15 in latest push. Note: when running storybook it says version 6.1.14 but in the console it correctly shows 6.1.15 as the version being used and the package.json and yarn.lock show 6.1.15